fsspec/implementations/local.py

```diff
@@ -350,7 +350,7 @@ def __getattr__(self, item):
 
     def __enter__(self):
         self._incontext = True
-        return self.f.__enter__()
+        return self
```

Comment:
Am I correct in assuming that this will solve 

> `open_file = fsspec.open(local_path) -> <OpenFile '/local/path.nc'>`
> ❌  doesn't work with context manager (TypeError: cannot pickle '_io.BufferedReader' object)

from https://github.com/fsspec/filesystem_spec/issues/579#issuecomment-1140377495

Reply:
Update: just re-ran my test and the answer is YES! ✅